actors/providers/postgresql/src/main/java/com/ea/orbit/actors/providers/postgresql/PostgreSQLStorageProvider.java

```diff
@@ -55,7 +55,7 @@
     private String username;
     private String password;
 
-    private Connection conn;
+    private Connection connection;
     private PreparedStatement insertState;
```

Comment:
general observation of the postgres implementation, I would consider removing the sync keywords and creating a new prepared statement every time we need to read / write and close it in a finally block..